npm/@bufbuild/buf/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@bufbuild/buf",
+  "description": "The buf CLI is a tool for working with Protocol Buffers.",
```

Comment:
This description is what shows up in a search on npmjs.com, with `npm info`, and it's used by most IDEs when looking at imports or project dependencies.

Maybe we should use that chance to consolidate the descriptions across the different ecosystems? 

[On scoop](https://scoop.sh/#/apps?q=buf&s=0&d=1&o=true), we use:

> A tool for working with Protocol Buffers APIs, offering a range of features not found in the standard protoc compiler.

On [aur](https://aur.archlinux.org/packages/buf), it's 

> A new way of working with Protocol Buffers

And our home page says [about the CLI](https://buf.build/product/cli/):

> The Buf CLI is a one stop shop for your local Protocol Buffers needs

I'm only raising this because we can't retroactively change this description without cutting a release.

IMO we should definitely have a README.md here, because that is what shows up on npmjs.com ([example](https://www.npmjs.com/package/@bufbuild/protoc-gen-es)), and we have to cut a release to add / change it as well. 

Our top-level README.md is actually pretty nice, so perhaps it is sufficient to link it here? If we do that, we have to update [this link](https://github.com/bufbuild/buf/blob/a1a4fbd9ddebcded1a3ef1db30dcba7180919dcc/README.md?plain=1#L155) (and perhaps others), which would otherwise not work as expected when rendered on npmjs.com.

Reply:
> Maybe we should use that chance to consolidate the descriptions across the different ecosystems?

All up for consolidation. I'll defer to @bufdev or @akshayjshah on this one.

> IMO we should definitely have a README.md here, because that is what shows up on npmjs.com ([example](https://www.npmjs.com/package/@bufbuild/protoc-gen-es)), and we have to cut a release to add / change it as well.
> 
> Our top-level README.md is actually pretty nice, so perhaps it is sufficient to link it here? If we do that, we have to update [this link](https://github.com/bufbuild/buf/blob/a1a4fbd9ddebcded1a3ef1db30dcba7180919dcc/README.md?plain=1#L155) (and perhaps others), which would otherwise not work as expected when rendered on npmjs.com.

Hmm not sure i know exactly what you mean, but I do agree we should have a better README.md. If we could reuse the top level readme, that would be ideal.

Reply:
I mean:

```sh
ln -s ../../../README.md npm/@bufbuild/buf
```

This would give you a link at npm/@bufbuild/buf/README.md pointing to README.md.

Not important.

But if we do use this README, we have to make sure all links are absolute.

Reply:
Ah right, with a symlink. Makes sense. I guess we'll lose that if it lives in it's own repo.

Reply:
Am not sure the symlink would have worked. Considering that we fetch binaries from the release page, we might just as well fetch the README.md from the repo?

Reply:
Sure, why not.